### PR TITLE
Remove swaps which are made on Harmony network

### PR DESCRIPTION
**Pull request by AndonMitev**
hide swaps which are on harmony network

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -38,6 +38,11 @@ const assets = {
     decimals: 6,
     explorer: 'https://etherscan.io/tx/',
   },
+  ONE: {
+    name: 'ONE',
+    decimals: 18,
+    explorer: 'https://etherscan.io/tx/',
```
Comment (kraikov) [inline src/config.js]: https://explorer.harmony.one/#/tx/

File: src/context/history/index.js [hunk 2]
```diff
@@ -52,7 +54,9 @@ export default function Provider({ children }) {
   useEffect(() => {
     function get() {
       getHistory().then((swaps) => {
-        update(swaps);
+        // Remove swaps which are made on Harmony network
+
+        update(swaps.filter((swap) => swap.network !== 'ONE'));
```
Comment (kraikov) [inline src/context/history/index.js]: Remove this.

**Review by kraikov — CHANGES_REQUESTED** [review]

File: src/config.js (hunk 1 above)
Comment (AndonMitev) [reply src/config.js]: Fixed

File: src/context/history/index.js (hunk 2 above)
Comment (AndonMitev) [reply src/context/history/index.js]: Fixed

**Pull request merged**